extensions/skins/conversation_v1/player.html

```diff
@@ -36,30 +34,22 @@
         cards, (b) the screen is wide, in which case both cards appear
         side-by-side.
       -->
-      <md-card class="conversation-skin-tutor-card conversation-skin-tutor-card-active"
-               ng-class="{'conversation-skin-left-card': isCurrentSupplementalCardNonempty() && canWindowFitTwoCards(), 'conversation-skin-has-subsidiary-cards': isCurrentSupplementalCardNonempty() && !canWindowFitTwoCards()}" ng-show="isPanelVisible(PANEL_TUTOR)">
-        <div class="conversation-skin-tutor-card-feedback-toggle-container" ng-if="activeCard.stateName && !isInPreviewMode">
-          <button class="conversation-skin-tutor-card-feedback-toggle"
```

Comment:
It just feels crowded at the top of the card if we leave it there, and it's confusing to have both card-level and exploration-level feedback; ideally, the exploration-level feedback should just include meta-data about which card the feedback was left from.

Re: the 'speak' button, I think that should go in the navbar, where the info and edit buttons are going to be, since the user should be toggling speaking for the entire exploration, rather than toggling speaking for each individual card. We can work on the design of that in a separate thread.